.vitepress/theme/components/AppContainer.vue

```diff
@@ -9,16 +8,17 @@ const props = defineProps<{
 
 <template>
   <div bg="zinc-100 dark:zinc-800" w-full flex rounded-xl p-4>
-    <div flex="~" max-h-40 max-w-40 justify-center>
+    <div flex="~ shrink-0" max-h-40 max-w-40 w="<sm:34%" justify-center>
```

Comment:
<img width="407" alt="image" src="https://github.com/nolebase/nolebase/assets/11081491/d9afdb9a-980d-46e3-a305-6591548945ba">


```suggestion
    <div flex="~" h-30 min-w-30 w-30 justify-center object-contain>
```

是不是直接这样就好了，其他的都不需要？

Reply:
设置成固定大小的话的确不需要了，我是想在尽量不变更原来的样式的情况下更改所以这样设定的。
另外在父级元素设定object-contain 对里面的是不生效的，里面元素是插槽传过来的所以也不能直接添加

Reply:
我理解你的意思了。

那在 Vue 编译后 slotted 选择器也依然会生效吗？Vue 的 template 和 slot 好像是不能和 Web Component 的混着用的。是这样的吗？

如果不能用的话，给 div 加一个 class 然后 CSS 里面选择器选中 .some-class img 这样直接改 object-fit 可以吗？

或者另一个思路是，让外面传进来的 img 自己配置这些选项会好吗？

Reply:
> 如果不能用的话，给 div 加一个 class 然后 CSS 里面选择器选中 .some-class img 这样直接改 object-fit 可以吗？

如果是在`style scoped`直接添加的话应该不行。用vue的插槽选择器的话就可以。就像我先前加的那样。只是将作用域更加限定到了图标那里。

> 或者另一个思路是，让外面传进来的 img 自己配置这些选项会好吗？

我是想尽量减轻使用负担的，所以更喜欢在组件内解决。哪怕实现的不是那么好。
毕竟开发一次，使用无数次。